Altis_Life.Altis/core/items/fn_boltcutter.sqf

```diff
@@ -6,48 +6,68 @@
+    hint format [localize "STR_Civ_NotEnoughCops", (LIFE_SETTINGS(getNumber, "minimum_cops"))];
 };
-if ((typeOf _building) == _vaultHouse && (nearestObject [_pos,"Land_Dome_Big_F"]) getVariable ["locked",true]) exitWith {hint localize "STR_ISTR_Bolt_Exploit"};
+if ((typeOf _building) == _vaultHouse && (nearestObject [_pos, "Land_Dome_Big_F"]) getVariable ["locked", true]) exitWith {hint localize "STR_ISTR_Bolt_Exploit"};
 if (isNil "life_boltcutter_uses") then {life_boltcutter_uses = 0;};
 
-_doors = FETCH_CONFIG2(getNumber,"CfgVehicles",(typeOf _building),"numberOfDoors");
+_doors = FETCH_CONFIG2(getNumber, "CfgVehicles", (typeOf _building), "numberOfDoors");
 _door = 0;
 //Find the nearest door
 for "_i" from 1 to _doors do {
-    _selPos = _building selectionPosition format ["Door_%1_trigger",_i];
+    _selPos = _building selectionPosition format ["Door_%1_trigger", _i];
     _worldSpace = _building modelToWorld _selPos;
         if (player distance _worldSpace < 2) exitWith {_door = _i;};
 };
 if (_door isEqualTo 0) exitWith {hint localize "STR_Cop_NotaDoor"}; //Not near a door to be broken into.
-if ((_building getVariable [format ["bis_disabled_Door_%1",_door],0]) isEqualTo 0) exitWith {hint localize "STR_House_Raid_DoorUnlocked"};
+if ((_building getVariable [format ["bis_disabled_Door_%1", _door], 0]) isEqualTo 0) exitWith {hint localize "STR_House_Raid_DoorUnlocked"};
 
-if ((nearestObject [_pos,"Land_Dome_Big_F"]) == _building || (nearestObject [_pos,_vaultHouse]) == _building) then {
-    [[1,2],"STR_ISTR_Bolt_AlertFed",true,[]] remoteExecCall ["life_fnc_broadcast",RCLIENT];
+if ((nearestObject [_pos, "Land_Dome_Big_F"]) == _building || (nearestObject [_pos, _vaultHouse]) == _building) then {
+    [[1, 2], "STR_ISTR_Bolt_AlertFed", true, []] remoteExecCall ["life_fnc_broadcast", RCLIENT];
 } else {
-    [0,"STR_ISTR_Bolt_AlertHouse",true,[profileName]] remoteExecCall ["life_fnc_broadcast",RCLIENT];
+    [0, "STR_ISTR_Bolt_AlertHouse", true, [profileName]] remoteExecCall ["life_fnc_broadcast", RCLIENT];
 };
 
+private _ownerUid = (_building getVariable ["house_owner", ["", ""]]) select 0;
+if (_ownerUid != "") then {_home = true};
+if (_home && (!([_ownerUid] call life_fnc_isUIDActive))) exitWith {hint localize "STR_ISTR_Bolt_Offline"};
+private _copsNeeded = LIFE_SETTINGS(getNumber, "copsHouseRaid");
+if (({side _x isEqualTo west} count playableUnits < _copsNeeded) && _home) exitWith {hint format [localize "STR_Civ_NotEnoughCops", _copsNeeded]};
+
 life_action_inUse = true;
+
+if (_home) then {
+    private _unitsToNotify = [];
+    {
+        if (_uid isEqualTo (getPlayerUID _x) || side _x isEqualTo west) then {_unitsToNotify pushBack _x};
+        if (_uid isEqualTo (getPlayerUID _x)) then {_ownerName = name _x};
+    } forEach playableUnits;
+    if (count _unitsToNotify isEqualTo 0) exitWith {};
+    [1, [_building, 60, "Mil_dot", "HOUSE ROBBERY IN PROGRESS"]] remoteExec ["life_fnc_markers", _unitsToNotify];
```

Comment:
See [this comment](https://github.com/AsYetUntitled/Framework/pull/73/files#r97927802).